blocks/autocompleters/README.md

```diff
@@ -0,0 +1,117 @@
+Autocompleters
+==============
+
+Autocompleters enable us to offer users options for completing text input. For example, Gutenberg includes a user autocompleter that provides a list of user names and completes a selection with a user mention like `@mary`.
```

Comment:
Can we add a reference to `@wordpress/components`? It contains the same content and will be difficult to keep in sync otherwise.

Reply:
I actually meant to remove that entirely after moving the content but missed doing so last night.

Reply:
Leaving a README with a reference to `@wordpress/components` seems good. Is there a better way than direct GitHub links to link to component READMEs?

Reply:
Unfortunately not yet :(